jrqs-app/src/java/ua/dp/isd/jrqs/ESearchBy.java

```diff
@@ -0,0 +1,37 @@
+/*
+ *  jRQS
+ *  Java Implementation of RQS
+ *  Copyright (C) Alexander Fuks 2013-2017
+ *  
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *  
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *  
+ *  You should have received a copy of the GNU General Public License along
+ *  with this program; if not, write to the Free Software Foundation, Inc.,
+ *  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *  
+ *  Alexander Fuks, hereby disclaims all copyright
+ *  interest in the program "jRQS"
+ *  (which makes passes at compilers)
+ *  written by Alexander Fuks.
+ * 
+ *  Alexander Fuks <mailto:Alexander.V.Fuks@gmail.com>, June 29, 2017.
+ */
+package ua.dp.isd.jrqs;
+
+public final class ESearchBy {
+    
+    public static enum SearchBy {
```

Comment:
You don't need to make ENUMs as a part of class. Or at lest not public access. Simply, if it is something related to one class, make it private, otherwise don't make it nested if several classes will have access to it.